flanker/.gitignore

```diff
@@ -0,0 +1 @@
+/build

```

Comment:
let's add to git ignore default files. like .idea. I think this link can help https://github.com/github/gitignore/blob/master/Android.gitignore

Reply:
This is already in the root `.gitignore`